src/config/index.ts

```diff
@@ -71,15 +71,17 @@ export const SIGN_OUT_PAGE = "sign-out";
 export const TRUST_DETAILS_PAGE = "trust-details";
 export const TRUST_INVOLVED_PAGE = "trust-involved";
 export const CONFIRM_OVERSEAS_ENTITY_DETAILS_PAGE = "confirm-overseas-entity-details";
+export const UPDATE_OVERSEAS_ENTITY_REVIEW_PAGE = "overseas-entity-review";
 
 // ROUTING PATHs
 export const LANDING_URL = "/register-an-overseas-entity";
 export const UPDATE_LANDING_URL = "/update-an-overseas-entity";
 
 export const REGISTER_AN_OVERSEAS_ENTITY_URL = LANDING_URL + "/";
 export const UPDATE_AN_OVERSEAS_ENTITY_URL = UPDATE_LANDING_URL + "/";
-export const INTERRUPT_CARD_URL = REGISTER_AN_OVERSEAS_ENTITY_URL + INTERRUPT_CARD_PAGE;
 export const OVERSEAS_ENTITY_QUERY_URL = UPDATE_AN_OVERSEAS_ENTITY_URL + OVERSEAS_ENTITY_QUERY_PAGE;
+export const CONFIRM_OVERSEAS_ENTITY_PROFILES_URL = UPDATE_AN_OVERSEAS_ENTITY_URL + CONFIRM_OVERSEAS_ENTITY_DETAILS_PAGE;
```

Comment:
I think we were planning to have all pages start with OVERSEAS_ENTITY, so this page will have to be renamed as OVERSEAS_ENTITY_CONFIRM_PROFILES_URL , or I can rename them as part of my PR , @AndreKramer7654 do you agree with the naming format

Reply:
Please rename as part of a different ticket